### PR TITLE
fix: auto remove unnecessary braces after remove unused imports

### PR DESCRIPTION
**Pull request by Young-Flash**
before
before

after
after

### CODE REVIEW

**Comment by Young-Flash** [comment]
could maintainers help review this when have time? 

cc @Veykril @lnicola @HKalbasi

**Review by lnicola — APPROVED** [review]
Looks good. I just wasn't sure that `ted` is the best API here, but we can probably merge it like this anyway. I'll just wait until tomorrow for a second review.

**Review by Veykril — APPROVED** [review]

**Pull request merged**
